client/src/actions/user.js

```diff
@@ -35,6 +35,7 @@ export function getUserInfo () {
             .get('http://localhost:8000/app/users/me/')
             .set('Authorization', 'Token ' + localStorage.getItem('token'))
             .end(function(err, res) {
+                // NOTE don't think '404' works
```

Comment:
Yeah, @hxkoh2 do you guys use 404 as an error status? Or does Django handle that by itself. Any advice on the statuses we should be checking? Or should we just use a catchall .err() maybe Annie.